integrations/amazon-bedrock.md

```diff
@@ -64,3 +65,32 @@ Output:
 ```shell
 'There is no definitive "best" American actor, as acting skill and talent a# re subjective. However, some of the most acclaimed and influential American act# ors include Tom Hanks, Daniel Day-Lewis, Denzel Washington, Meryl Streep, Rober# t De Niro, Al Pacino, Marlon Brando, Jack Nicholson, Leonardo DiCaprio and John# ny Depp. Choosing a single "best" actor comes down to personal preference.'
 ```
+
+### AmazonBedrockChatGenerator
+
+To use this integration for chat models, initialize a `AmazonBedrockChatGenerator` with the model name and aws credentials:
+
+Currently, the following models are supported: 
+- Anthropic's Claude
+- Meta's Llama 2
+
+
+```python
+from haystack_integrations.components.generators.amazon_bedrock import AmazonBedrockChatGenerator
```

Comment:
I'm getting an error here --
```

ImportError: cannot import name 'AmazonBedrockChatGenerator' from 'haystack_integrations.components.generators.amazon_bedrock' (/usr/local/lib/python3.10/dist-packages/haystack_integrations/components/generators/amazon_bedrock/__init__.py)
```

probably doing something obvious wrong - I ran `pip install amazon-bedrock-haystack` and `haystack-ai` but is there another depencency that's missing or something?

Reply:
@annthurium haven't published pypi release yet. Waiting for docs to be finished by Daria etc
But please try from main directly and TIA 🙏

Reply:
@vblagoje is there a way to install a haystack integration from main other than git cloning the whole repo and trying to run it locally? Sorry, I haven't had to do this yet and it's not documented anywhere I could find

Reply:
I'm not 100% - I always run it directly from a clone of haystack-core-integrations project where I can change branches. Perhaps that is the best for dev setup...